js/src/offcanvas.js

```diff
@@ -121,6 +122,10 @@ class Offcanvas extends BaseComponent {
     this._element.classList.add(CLASS_NAME_SHOW)
 
     const completeCallBack = () => {
+      if (!this._config.scroll) {
+        this._element.focus()
+      }
```

Comment:
can this got on line 117 after activation? (or maybe on activation the focusTrap to focus on trapElement?)

Reply:
I've done it this way to match the initial focus behavior of the modal component. There, it focuses the element when the show transition completes. If I were to move this to line 117, the focus would jump to Offcanvas before it has finished transitioning to shown.

Reply:
Hmmm, maybe the opposite? (bring `this._focustrap.activate()` on line 126)